index.html

```diff
@@ -5824,6 +5824,13 @@ <h3>LoadDocumentCallback</h3>
         <li>Set <var>documentUrl</var> to the location of the retrieved resource
           considering redirections (exclusive of HTTP status <code>303</code> "See Other" redirects
           as discussed in [[?cooluris]]).</li>
+        <li>If the retrieved resource's <a>Content-Type</a> is not <code>application/ld+json</code>
+          nor any media type with a <code>+json</code> suffix as defined in [[RFC6839]],
+          and the response has an HTTP Link Header [[RFC8259]] using the <code>alternate</code> link relation
```

Comment:
RFC8259 describe the JSON format, not the Link Header... Note that this error is present in other parts of the document (I imagine you just copy-pasted it from there).

Reply:
Thanks, I’ll fix that. Also, this should limit application/json, not just ld+json and other +json types.

Reply:
Web Linking is [RFC8288](https://tools.ietf.org/html/rfc8288) (confusingly close looking 😉).